OTAnalytics/plugin_ui/customtkinter_gui/dummy_viewmodel.py

```diff
@@ -1850,5 +1850,9 @@ def set_frame_track_statistics(self, frame: AbstractFrameTrackStatistics) -> Non
         self._frame_track_statistics = frame
 
     def update_track_statistics(self, _: EventRepositoryEvent) -> None:
+        if self._frame_track_statistics is None:
+            raise MissingInjectedInstanceError(
+                type(self._frame_track_statistics).__name__
```

Comment:
This is not going to work as expected. This is equivalent to `type(None).__name__`. It might be a relict of one of the other frames where we missed to change the error message.

Reply:
I refactored this introducing properties for each instance that might not be injected yet. Raising the `MissingInjectedInstanceError` is now handled in the respective properties.